### PR TITLE
Add glGetAttribLocation() and more glUniforms()

### PR DESCRIPTION
**Pull request by segfault87**
This PR includes addition of following OpenGL calls:

* glGetAttribLocation
* glUniform

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -395,6 +401,42 @@ pub trait Context {
 
     unsafe fn uniform_1_i32(&self, location: Option<Self::UniformLocation>, x: i32);
 
+    unsafe fn uniform_2_i32(&self, location: Option<Self::UniformLocation>, x: i32, y: i32);
+
+    unsafe fn uniform_3_i32(&self, location: Option<Self::UniformLocation>, x: i32, y: i32, z: i32);
+
+    unsafe fn uniform_4_i32(&self, location: Option<Self::UniformLocation>, x: i32, y: i32, z: i32, w: i32);
+
+    unsafe fn uniform_1_i32_v(&self, location: Option<Self::UniformLocation>, v: &mut [i32; 1]);
```
Comment (grovesNL) [inline src/lib.rs]: I've been using `_slice` instead of `_v` so far, what do you think?

Comment (grovesNL) [inline src/lib.rs]: Is `&mut` (instead of `&`) still required on recent versions of web-sys? It looks like `uniform_1_f32_v` doesn't require it

Comment (segfault87) [reply src/lib.rs]: `_slice` sounds more natural! I'll fix them.

Comment (segfault87) [reply src/lib.rs]: Oh, I've been looking on the old version of web-sys... I'll fix them.

Comment (segfault87) [reply src/lib.rs]: https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.WebGlRenderingContext.html#method.uniform1iv_with_i32_array

My mistake, `&mut`s are still there. I think `mut` keywords are there for no reason and should be removed. We have to keep the keywords there until web-sys is fixed.

Comment (grovesNL) [reply src/lib.rs]: Sounds good

**Comment by grovesNL** [comment]
Thank you! I just rebased this (to remove the merge commit) and merged into master

**Pull request closed without merging**